tests/src/reflection/GenericAttribute/GenericAttributeTests.cs

```diff
@@ -0,0 +1,183 @@
+using System;
+using System.Reflection;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        new MultiAttribute<string>("a");
```

Comment:
This line looks like an accidental leftover.

Reply:
sorry